docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -14,52 +14,44 @@ that Hotspots are honestly representing their location and the wireless network
 creating from that location.
 
 LoRaWAN Hotspots utilize a PoC method that benefits from Hotspots being used as senders ("Beacon")
-and receivers ("Witness") of PoC activity, you can read more here:
+and receivers ("Witness") of PoC activity, read more here:
 [Proof of Coverage](/blockchain/proof-of-coverage)
 
 While all current 5G Hotspot models _also_ participate in the LoRaWAN network and PoC, the CBRS
 Small Cell and 5G aspects of a 5G Hotspot require a new approach to validating PoC.
 
-**One Way Traffic**
+:::info **One Way Traffic**
 
 CBRS Small Cell Radios can only operate as transmitters, they are not able to receive incoming data
 and thus can not "hear" one another directly.
 
-## MOBILE PoC Roadmap
-
-:::info Current Progress
-
-We are currently in Stage 1, Genesis Phase 2:
-[Time Period Triggered Rewards with Minimum Delay](/5g-on-helium/mobile-poc#genesis-phase-2-time-period-triggered-rewards-with-minimum-delay)
-
 :::
 
+## MOBILE PoC Roadmap
+
 - **Stage 1: ** Genesis Rewards
-  - _Genesis Phase 1_:
-    [Time Period Triggered Rewards](/5g-on-helium/mobile-poc#genesis-phase-1-time-period-triggered-rewards)
-  - _Genesis Phase 2_:
-    [Time Period Triggered Rewards with Minimum Delay](/5g-on-helium/mobile-poc#genesis-phase-2-time-period-triggered-rewards-with-minimum-delay)
-  - _Genesis Phase 3_:
-    [Speed Test and Quality of Service](/5g-on-helium/mobile-poc#genesis-phase-3-speed-tests-and-quality-of-service)
-  - _Genesis Phase 4_:
-    [Heartbeats and Consistency of Service](/5g-on-helium/mobile-poc#genesis-phase-4-heartbeats-and-reliability-of-service)
-- **Stage 2: ** 5G Mappers Proof of Coverage
-- **Stage 3: ** Additional Methods of Proof of Coverage
+  - _Current Implementation_:
+    [Speed Test and Quality of Service](/5g-on-helium/mobile-poc#speed-tests-and-quality-of-service)
+  - _Work in Progress_:
+    [Heartbeats and Consistency of Service](/5g-on-helium/mobile-poc#heartbeats-and-reliability-of-service)
+- **Stage 2: ** [5G Mappers Proof of Coverage](/5g-on-helium/mobile-poc#mobile-mappers)
+- **Stage 3: ** [Additional Methods of Proof of Coverage](/5g-on-helium/mobile-poc#future-roadmap)
 
 ---
 
-## Current MOBILE PoC Reward Elegibility
+## PoC Reward Elegibility
```

Comment:
Not sure why this section is needed. It currently says that speed tests are required, and next section says that Speed Tests will be required.